# HG changeset patch
# User Tatsuro Matsuoka <tmacchant@yahoo.co.jp>
# Date 1428728871 -32400
#      Sat Apr 11 14:07:51 2015 +0900
# Node ID d7f7ce99532b2b15170bf0fbd824c58e7c5dd007
# Parent  db30302bedc30e2e10f48a9a7dcedc7fcd641ade
Update jp_JP.ts (Japanese Translation).

* jp_JP.ts: Update translation.

diff --git a/libgui/languages/ja_JP.ts b/libgui/languages/ja_JP.ts
--- a/libgui/languages/ja_JP.ts
+++ b/libgui/languages/ja_JP.ts
@@ -1141,82 +1141,82 @@ is already open in the editor</source>
     <message>
         <location line="+5"/>
         <source>&amp;Show Completion List</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Format</source>
-        <translation type="unfinished"></translation>
+        <translation>フォーマット (&amp;)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Uppercase Selection</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Lowercase Selection</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+12"/>
         <source>&amp;Indent</source>
-        <translation type="unfinished"></translation>
+        <translation>インデント (&amp;I)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Unindent</source>
-        <translation type="unfinished"></translation>
+        <translation>アンインデント (&amp;U)</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Convert Line Endings to &amp;Windows (CRLF)</source>
-        <translation type="unfinished"></translation>
+        <translation>行末文字を Windows 仕様にする (CRLF) (&amp;W)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Convert Line Endings to &amp;Unix (LF)</source>
-        <translation type="unfinished"></translation>
+        <translation>行末文字を Unix 仕様にする (LF) (&amp;U)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Convert Line Endings to &amp;Mac (CR)</source>
-        <translation type="unfinished"></translation>
+        <translation>行末文字を Mac 仕様にする (CR) (&amp;U)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Navi&amp;gation</source>
-        <translation type="unfinished"></translation>
+        <translation>ナビゲーション (&amp;g)</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Move to Matching Brace</source>
-        <translation type="unfinished"></translation>
+        <translation>対応する括弧に移動</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Select to Matching Brace</source>
-        <translation type="unfinished"></translation>
+        <translation>対応する括弧を選択</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>&amp;Preferences...</source>
         <translation>設定 (&amp;P) ...</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Styles Preferences...</source>
         <translation>スタイルの設定 (&amp;S) ...</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>&amp;View</source>
-        <translation type="unfinished"></translation>
+        <translation>表示 (&amp;V)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Editor</source>
         <translation>エディタ (&amp;E)</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1318,17 +1318,17 @@ for write: %2.</source>
     <message>
         <location line="+36"/>
         <source>&amp;Edit</source>
         <translation>編集 (&amp;E)</translation>
     </message>
     <message>
         <location line="+138"/>
         <source>Show &amp;Whitespace Characters</source>
-        <translation type="unfinished"></translation>
+        <translation>ホワイトスペースを表示 (&amp;W)</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>&amp;Debug</source>
         <translation>デバッグ (&amp;D)</translation>
     </message>
     <message>
         <location line="+21"/>
@@ -2667,29 +2667,29 @@ Octave GUI はオープンできませんでした．</translation>
         <location line="+2"/>
         <source>Active</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="-159"/>
         <location filename="../src/ui-settings-dialog.h" line="-11"/>
         <source>Small</source>
-        <translation type="unfinished"></translation>
+        <translation>小さい</translation>
     </message>
     <message>
         <location line="+17"/>
         <location filename="../src/ui-settings-dialog.h" line="+2"/>
         <source>Large</source>
-        <translation type="unfinished"></translation>
+        <translation>大きい</translation>
     </message>
     <message>
         <location line="+168"/>
         <location filename="../src/ui-settings-dialog.h" line="+10"/>
         <source>Custom style</source>
-        <translation type="unfinished"></translation>
+        <translation>カスタムスタイル</translation>
     </message>
     <message>
         <location line="+13"/>
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <source>Background inactive</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -2705,17 +2705,17 @@ Octave GUI はオープンできませんでした．</translation>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+41"/>
         <location line="+1392"/>
         <location filename="../src/ui-settings-dialog.h" line="+2"/>
         <location line="+82"/>
         <source>Startup path</source>
-        <translation type="unfinished"></translation>
+        <translation>起動パス</translation>
     </message>
     <message>
         <location line="-1379"/>
         <location line="+1409"/>
         <location filename="../src/ui-settings-dialog.h" line="-81"/>
         <location line="+83"/>
         <source>Browse</source>
         <translation type="unfinished"></translation>
@@ -2737,23 +2737,23 @@ Octave GUI はオープンできませんでした．</translation>
         <location filename="../src/ui-settings-dialog.h" line="+5"/>
         <source>Color</source>
         <translation>色</translation>
     </message>
     <message>
         <location line="+40"/>
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <source>Tab width min.</source>
-        <translation type="unfinished"></translation>
+        <translation>タブ幅の最小値</translation>
     </message>
     <message>
         <location line="+64"/>
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <source>max.</source>
-        <translation type="unfinished"></translation>
+        <translation>最大値</translation>
     </message>
     <message>
         <location line="+31"/>
         <location filename="../src/ui-settings-dialog.h" line="+2"/>
         <source>This works well for monospaced fonts. The line is drawn at a position based on the width of a space character in the default font. It may not work very well if styles use proportional fonts or if varied font sizes or bold, italic and normal texts are used.</source>
         <translation>本機能は等幅フォントで正常動作します．既定フォントのスペースのキャラクターの幅で行は表示されます．スタイルがプロポーショナルフォントを使う場合や可変フォントサイズ，ボールド，イタリックフォントとノーマルテクストが使用された場合正常に動作しません．</translation>
     </message>
     <message>
@@ -3163,17 +3163,17 @@ Octave GUI はオープンできませんでした．</translation>
         <location filename="../src/ui-settings-dialog.h" line="-5"/>
         <source>Icon size</source>
         <translation>アイコンのサイズ</translation>
     </message>
     <message>
         <location line="+26"/>
         <location filename="../src/ui-settings-dialog.h" line="+3"/>
         <source>Normal</source>
-        <translation type="unfinished"></translation>
+        <translation>標準</translation>
     </message>
     <message>
         <location line="+1195"/>
         <location filename="../src/ui-settings-dialog.h" line="+72"/>
         <source>File handling</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -3219,23 +3219,23 @@ Octave GUI はオープンできませんでした．</translation>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+31"/>
         <location line="+28"/>
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <location line="+4"/>
         <source>Use this set</source>
-        <translation type="unfinished"></translation>
+        <translation>このセットを使用する</translation>
     </message>
     <message>
         <location line="-21"/>
         <location filename="../src/ui-settings-dialog.h" line="-3"/>
         <source>Set 1:</source>
-        <translation type="unfinished"></translation>
+        <translation>セット 1</translation>
     </message>
     <message>
         <location line="+7"/>
         <location line="+37"/>
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <location line="+3"/>
         <source>Export</source>
         <translation>エクスポート</translation>
@@ -3247,17 +3247,17 @@ Octave GUI はオープンできませんでした．</translation>
         <location line="+3"/>
         <source>Import</source>
         <translation>インポート</translation>
     </message>
     <message>
         <location line="+7"/>
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <source>Set 2:</source>
-        <translation type="unfinished"></translation>
+        <translation>セット 2</translation>
     </message>
     <message>
         <location line="+86"/>
         <location filename="../src/ui-settings-dialog.h" line="+7"/>
         <source>Action</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -3325,17 +3325,17 @@ Octave GUI はオープンできませんでした．</translation>
         <location line="+4"/>
         <location line="+561"/>
         <source>System setting</source>
         <translation>システム設定</translation>
     </message>
     <message>
         <location line="-355"/>
         <source>IBeam Cursor</source>
-        <translation>IBeam カーソル标</translation>
+        <translation>IBeam カーソル</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Block Cursor</source>
         <translation>ブロックカーソル</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -3859,17 +3859,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Convert Line Endings to Mac</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+25"/>
         <source>Show Whitespace Characters</source>
-        <translation type="unfinished"></translation>
+        <translation>ホワイトスペースの表示</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Line Endings</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+1"/>
@@ -3974,17 +3974,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="-23"/>
         <source>News</source>
         <translation>ニュース</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>View</source>
-        <translation>ビュー</translation>
+        <translation>表示</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Run</source>
         <translation>実行</translation>
     </message>
     <message>
         <location line="+124"/>
